.travis.yml

```diff
@@ -24,6 +24,14 @@ before_install:
 
 install: true
 
+before_script:
+- |
+  if [[ ${TRAVIS_PHP_VERSION:0:2} == "7." ]]; then
+    composer global require "phpunit/phpunit=5.7.*"
+  elif [[ ${TRAVIS_PHP_VERSION:0:3} != "5.2" ]]; then
```

Comment:
I just copied the core patch (https://core.trac.wordpress.org/changeset/40257). I guess since we are only testing 7.x right now, we don't need the conditional; we can just load phpunit 5.7.